source/manual/2nd-line-drills.html.md

```diff
@@ -81,9 +81,47 @@ See "[If Slack is unavailable](https://docs.google.com/document/d/144y8c2Ly-kG3J
 In preparation for a spike in traffic, you can increase the number of unicorn workers for an app.
 See [established connections exceeded](/manual/alerts/established-connections-exceed.html) for details.
 
-Pick an application and drill scaling up the number of workers - see [example](https://github.com/alphagov/govuk-puppet/pull/11194).
+Pick an application and drill scaling up the number of workers.
```

Comment:
See comment below - we'll move the detailed docs into its own file. I'd then make this file all about the drill specific considerations, e.g.

```md
In preparation for a spike in traffic, you can increase the number of unicorn workers for an app.

Pick an application (e.g. `smartanswers`) and drill scaling up the number of workers, by following [link](LINK).

Note that you'll need to edit the `integration.yaml` file, not the `production.yaml` file as described in the docs above. And instead of merging to `main`, just build your branch directly to Integration. When you're done drilling, re-deploy the previous release and delete your branch.
```

Reply:
Thanks Chris, good suggestion. I've made these changes.